src/main/java/com/o19s/es/ltr/feature/store/StoredFeatureSet.java

```diff
@@ -97,16 +106,32 @@ public StoredFeatureSet(String name, List<StoredFeature> features) {
                         "feature names must be unique in a set.");
             }
         }
+
+        this.derivedFeatures = derivedFeatures;
+        derivedFeatureMap = new HashMap<>();
+        for (StoredDerivedFeature feature : derivedFeatures) {
+            ordinal++;
```

Comment:
this ordinal may not valid when calling `feature(int ordinal)` ould that help to store feature+derived in the same List?